x-pack/plugins/apm/common/constants.ts

```diff
@@ -38,9 +38,9 @@ export const ERROR_GROUP_ID = 'error.grouping_key';
 export const ERROR_CULPRIT = 'error.culprit';
 export const ERROR_LOG_MESSAGE = 'error.log.message';
 export const ERROR_LOG_STACKTRACE = 'error.log.stacktrace';
-export const ERROR_EXC_MESSAGE = 'error.exception.message';
-export const ERROR_EXC_STACKTRACE = 'error.exception.stacktrace';
-export const ERROR_EXC_HANDLED = 'error.exception.handled';
+export const ERROR_EXC_MESSAGE = 'error.exception.message'; // only to be used in es queries, since error.exception is now an array
+export const ERROR_EXC_STACKTRACE = 'error.exception.stacktrace'; // only to be used in es queries, since error.exception is now an array
+export const ERROR_EXC_HANDLED = 'error.exception.handled'; // only to be used in es queries, since error.exception is now an array
```

Comment:
This is a little weird. Not sure if I should keep these three constants, since they might be "abused".
The correct way to eg. `message` is: 
```js
const firstMessage = error.exception[0].message
```
or
```js
const messages = error.exception.map(e => e.message)
```

However, getting `message`, `stacktrace` and `handles` in from ES query:

```js
{
  _source: [
    'error.exception.message',
    'error.exception.stacktrace',
    'error.exception.handled',
  ];
}
```

Reply:
Didn't I see conversation between you and @ogupte about moving away from these ES constants and just using the path values directly in the query code?

Reply:
I tried moving away from them, but as I was going over the changes with APM Server folks it proved surprisingly hard to ensure that we were using the new ECS fields, since the strings were spread all over the place. 

We should definitely not use the constants like we did before `_.get(transaction, SERVICE_NAME)` but when we just need the `SERVICE_NAME` string in the ES query, I think it still makes sense.
Thoughts?

Reply:
I'm fine with either way, just remembered seeing that conversation. Maybe we can rename this file to "esQueryConstants" or "esQueryPaths" or something like that, to help describe what they are and possibly avoid using them as object accessors, which is where the array access problems would exist I think?